Libraries/LibJS/Token.cpp

```diff
@@ -27,6 +27,7 @@
 #include "Token.h"
 #include <AK/Assertions.h>
 #include <AK/StringBuilder.h>
+#include <AK/Utf32View.h>
```

Comment:
Hmm, how is this used here?

Reply:
It's used in `StringBuilder.append(Utf32View)`:

https://github.com/SerenityOS/serenity/blob/096fb691673d4e6c71173cb0950912b12f1f5b99/Libraries/LibJS/Token.cpp#L175-L177

Reply:
Aha, got it! Harder to spot with just the `{ }` :)